ash/src/entry.rs

```diff
@@ -54,33 +62,28 @@ pub enum InstanceError {
 
 impl fmt::Display for InstanceError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "InstanceError::{:?}", self)
-    }
-}
-
-impl Error for InstanceError {
-    fn description(&self) -> &str {
-        "InstanceError"
-    }
-
-    fn cause(&self) -> Option<&Error> {
-        if let &InstanceError::VkError(ref err) = self {
-            return err.source();
+        match self {
+            InstanceError::LoadError(e) => write!(f, "{}", e.join("; ")),
+            InstanceError::VkError(e) => write!(f, "{}", e),
         }
-        None
     }
 }
 
+impl Error for InstanceError {}
+
 #[allow(non_camel_case_types)]
 pub trait EntryV1_0 {
     type Instance;
     fn fp_v1_0(&self) -> &vk::EntryFnV1_0;
     fn static_fn(&self) -> &vk::StaticFn;
+    #[doc = "<https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkCreateInstance.html>"]
```

Comment:
I wish there would be a link to the newest docs like `vulkan/specs/newest`, maybe we should open an issue for this.